src/eve-server/market/MarketDB.cpp

```diff
@@ -343,12 +343,11 @@ uint32 MarketDB::StoreOrder(Market::SaveData &data) {
     return orderID;
 }
 
-PyRep *MarketDB::GetTransactions(uint32 clientID, Market::TxData& data) {
-    //transactionID, transactionDate, typeID, keyID, quantity, price,
-    //  transactionType, clientID, regionID, stationID, corpTransaction, characterID
+// Retrieves the market transactions owned by the current `characterID`.
+PyRep *MarketDB::GetTransactions(uint32 characterID, Market::TxData& data) {
     std::string typeID = "";
     if (data.typeID) {
-        typeID = "AND typeID=";
+        typeID = "typeID=";
```

Comment:
Removed the `AND` simply because the location of `typeID` in the below SQL query was changed such that it doesn't need the `AND` here more.